Bottom: 3722e75418364ab0bcf5c256cc3fbc6fa2990010
Top:    01e7b810cb6c586d381288539a3ebb43971f034f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 22:24:44 +0000

Refresh of adjust-mask-api-to-include

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index f73b1b7..290515f 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1740,7 +1740,7 @@ GhidDrawMask (int side, BoxType * screen)
       hid_draw_set_color (Output.pmGC, "erase");
     }
 
-  hid_draw_use_mask (Output.fgGC, HID_MASK_CLEAR);
+  hid_draw_use_mask (&ghid_graphics, HID_MASK_CLEAR);
 
   info.layer = Layer;
   info.drawn_area = screen;
@@ -1753,7 +1753,7 @@ GhidDrawMask (int side, BoxType * screen)
   r_search (PCB->Data->via_tree, screen, NULL, clearPin_callback_solid, NULL);
   r_search (PCB->Data->pad_tree, screen, NULL, clearPad_callback_solid, &side);
 
-  hid_draw_use_mask (Output.fgGC, HID_MASK_AFTER);
+  hid_draw_use_mask (&ghid_graphics, HID_MASK_AFTER);
   hid_draw_set_color (out->fgGC, PCB->MaskColor);
   ghid_set_alpha_mult (out->fgGC, thin ? 0.35 : 1.0);
 
@@ -1812,7 +1812,7 @@ GhidDrawMask (int side, BoxType * screen)
 #endif
   hidgl_shader_activate (circular_program);
 
-  hid_draw_use_mask (out->fgGC, HID_MASK_OFF);
+  hid_draw_use_mask (&ghid_graphics, HID_MASK_OFF);
 
 //  first_run = false;
 }
